dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/OrderAndPaginationExporterTest.java

```diff
@@ -107,6 +111,9 @@ void setUp() {
     // needed as some tests are run using another user (injectSecurityContext) while most tests
     // expect to be run by admin
     injectAdminUser();
+
+    operationParamsBuilder = EventOperationParams.builder();
```

Comment:
This test class is to test ordering/pagination of all enitites (right now its TE, EV).

So this should then probably be `eventOperationParamsBuilder`. We can shorten this to `eventParamsBuilder` IMHO